docs/developer-guide.rst

```diff
@@ -587,10 +591,10 @@ Inside your main plugin directory itself:
 			returnInternalIssues: true
 		}
 	};
-	describe ('Rule foo: Acceptances', function () {
-		it ('should accept some stuff and reject other stuff', function (done) {
-			var code = 'contract BlueBerry { function foo () {} }';
-			var errors = Solium.lint (code, config);
+	describe ('Rule foo: Acceptances', () => {
+		it ('should accept some stuff and reject other stuff', done => {
+			const code = 'contract BlueBerry { function foo () {} }';
```

Comment:
I know this is unsolicited but I'm eager to learn and found this PR through Gitcoin.  So, assuming you're willing to humor me; shouldn't `function foo ()` be changed to `constructor()`?